rmgpy/data/kinetics/kineticsTest.py

```diff
@@ -26,7 +28,386 @@ def testLoadFamilies(self):
+            """,
+            """
+            multiplicity 2
+            1 C u1 p0 c0 i13 {2,S} {3,S} {4,S}
+            2 H u0 p0 c0 {1,S}
+            3 H u0 p0 c0 {1,S}
+            4 H u0 p0 c0 {1,S}
+            """
+        ]
+
+        self.compare_degeneracy_of_reaction(adj_lists,rxn_family_str,correct_degeneracy)
+        
+    def test_degeneracy_for_ethyl_ethyl_disproportionation(self):
+        """Test that the proper degeneracy is calculated for ethyl + ethyl disproportionation"""
+
+        correct_degeneracy = 6
+        rxn_family_str = 'Disproportionation'
+        adj_lists = [
+            """
+            multiplicity 2
+            1 C u0 p0 c0 {2,S} {5,S} {6,S} {7,S}
+            2 C u1 p0 c0 {1,S} {3,S} {4,S}
+            3 H u0 p0 c0 {2,S}
+            4 H u0 p0 c0 {2,S}
+            5 H u0 p0 c0 {1,S}
+            6 H u0 p0 c0 {1,S}
+            7 H u0 p0 c0 {1,S}
+            """,
+            """
+            multiplicity 2
+            1 C u0 p0 c0 {2,S} {5,S} {6,S} {7,S}
+            2 C u1 p0 c0 {1,S} {3,S} {4,S}
+            3 H u0 p0 c0 {2,S}
+            4 H u0 p0 c0 {2,S}
+            5 H u0 p0 c0 {1,S}
+            6 H u0 p0 c0 {1,S}
+            7 H u0 p0 c0 {1,S}
+            """
+        ]
+
+        self.compare_degeneracy_of_reaction(adj_lists,rxn_family_str,correct_degeneracy)
+        
+    def test_degeneracy_for_ethyl_labeled_ethyl_disproportionation(self):
+        """Test that the proper degeneracy is calculated for ethyl + labeled ethyl disproportionation"""
+
+        correct_degeneracy = 3
+        rxn_family_str = 'Disproportionation'
+        adj_lists = [
+            """
+            multiplicity 2
+            1 C u0 p0 c0 i13 {2,S} {5,S} {6,S} {7,S}
+            2 C u1 p0 c0 {1,S} {3,S} {4,S}
+            3 H u0 p0 c0 {2,S}
+            4 H u0 p0 c0 {2,S}
+            5 H u0 p0 c0 {1,S}
+            6 H u0 p0 c0 {1,S}
+            7 H u0 p0 c0 {1,S}
+            """,
+            """
+            multiplicity 2
+            1 C u0 p0 c0 {2,S} {5,S} {6,S} {7,S}
+            2 C u1 p0 c0 {1,S} {3,S} {4,S}
+            3 H u0 p0 c0 {2,S}
+            4 H u0 p0 c0 {2,S}
+            5 H u0 p0 c0 {1,S}
+            6 H u0 p0 c0 {1,S}
+            7 H u0 p0 c0 {1,S}
+            """
+        ]
+        expected_products = 2
+        self.compare_degeneracy_of_reaction(adj_lists,rxn_family_str,correct_degeneracy * expected_products,expected_products)
+        
+    def compare_degeneracy_of_reaction(self, reactants_adj_list, 
+                                       rxn_family_str, 
+                                       correct_degeneracy_value,
+                                       num_products = 1):
+        """
+        input a list of adjacency lists (of reactants), 
+        the reaction family name,
+        and the correct degeneracy value
+        """
+        
+        degeneracy, reaction = self.find_reaction_degeneracy(reactants_adj_list,rxn_family_str,
```

Comment:
I can see how these terms are confusing.

`degeneracy` : the degeneracy value that RMG found
`correct_degeneracy` : the real degeneracy of the reaction
`correct_degeneracy_value`: the total number of degenerate reactions which should be found when generateReactions is called. 

I plan to rename `degeneracy` to `found_degeneracy`. I could refactor the methods to avoid `correct_degeneracy_value` all together or rename it to `expected_degenerate_products`.